sass/style.scss

```diff
@@ -90,10 +90,13 @@ hero {
   height: 252px;
   flex-shrink: 0;
   font-family: "Fjalla One", sans-serif;
+  
 
   .name,
   .description {
     font-size: 66px;
+    margin: 21px 0;
```

Comment:
Tutaj również stosujemy rem/em.
Pamiętaj żeby najlepiej dostosować to w taki sposób żeby skok był co 0.25
W tym przypadku  (przy założeniu że 1rem = 16px) 
To 21 px -> 1,3125rem -> zaokrąglamy w dół do 1.25rem